docs/UserGuide.adoc

```diff
@@ -75,6 +75,41 @@ You can use `entrylist` and `orderlist` to switch between the two lists.
 
 Person List Panel, Center Panel and Right Panel will be explained in greater detail in <<Features>>.
 
+=== Walkthrough
+Here's a simple walkthrough of Contact Sails.
+
+1) There are 6 contacts and 4 orders that are already pre-loaded in ContactSails to help you to get familiarised
+with the commands and features found in ContactSails. Click any contact on the left will `select` the contact,
+which will show you the contact's information on the center panel. For instance, clicking on
+Alex Yeoh on the left shows the picture below.
+
+image::SelectContactProcess.png[width="790"]
+
+2) Let us start by clearing these example contacts. Type `clear` on the command box at the top.
+Now, you should see an empty screen like the picture below:
+
+image::Empty.png[width = "790"]
+
+3) Let us add our first contact now. Type `add n/Mary p/96628855 e/mary@example.com a/Mary Road, block 1, #12-06, g/SoC pr/videoGame`
+in the command box and press enter. Click on the contact Mary and you should see this picture:
+
+image::ExampleMary.png[width = "790"]
```

Comment:
The pictures in the `images` folder have the `.PNG` extension, so the pictures won't show up in the UserGuide unless the extensions here are updated.